### PR TITLE
added card carousel (Home page)

### PR DESCRIPTION
**Pull request by luke-buttifant**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
| [**102**](https://github.com/Web-Dev-Path/web-dev-path/issues/102) |

#### Have you updated the CHANGELOG.md file? If not, please do it.
Changelog has been updated. 

#### What is this change?
Added a card carousel to the home page. 

#### Were there any complications while making this change?
No!

#### How did you verify this change?
I have tested various different screen sizes and there is also no console errors. 

#### When should this be merged?
After review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,3 +37,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - installed yarn
 - normalized buttons' and links' styling
 - updated `package.json` commands for macOs
+
+### changed
+
+- Prompt cards are now swipable. (Home page)
```
Comment (cherylli) [inline CHANGELOG.md:43]: maybe better to mention cards under which section or component. Should it be under #Added? a new swipable feature?

File: components/CardsColumns.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import Card from './Card';
 import Container from './Container';
+import { Swiper, SwiperSlide } from 'swiper/react';
```
Comment (cherylli) [inline components/CardsColumns.js:3]: The import order convention is usually node_module public module -> private modules -> local files

File: components/CardsColumns.js [hunk 3]
```diff
@@ -11,22 +13,46 @@ export default function CardsColumns({
...
-              content={content[index]}
-              link={links[index]}
-              linkText={linkText[index]}
-              key={index}
-            />
-          );
-        })}
-      </div>
-    </Container>
+    <>
```
Comment (cherylli) [inline components/CardsColumns.js:16]: this <></> may not be necessary.

**Review by cherylli — COMMENTED** [review]
Works perfectly! Just a few comments

General comments about the PR 
Please fill in "IssueNumber"
And assign yourself to the PR, and probably put tags in

There's no indication of more cards on the right side when viewing, so users probably won't notice it's a slider and just keep scrolling down.
[](https://i.imgur.com/cUqdJVK.png)

File: CHANGELOG.md (hunk 1 above)
Comment (luke-buttifant) [reply CHANGELOG.md]: Ok sure that makes more sense!

File: components/CardsColumns.js (hunk 2 above)
Comment (luke-buttifant) [reply components/CardsColumns.js]: Ok no problem 👍🏼

**Comment by cherylli** [comment]
Hi, I've just noticed the slider is still active at up to 1322 px width, but it should not be showing on screens > 769px,  as per requirement. 

I would decrease the gap between the dots and cards a little as they are off screen on my mobile.

**Comment by luke-buttifant** [comment]
> Hi, I've just noticed the slider is still active at up to 1322 px width, but it should not be showing on screens > 769px, as per requirement.
> 
> I would decrease the gap between the dots and cards a little as they are off screen on my mobile.

To have 3 cards showing at 769px caused them to bunch up, like this: 

image

So I played with the breakpoints and how many cards were showing to make it more seamless. The other alternative would be to make the cards/margin smaller so that they can all fit at 769px. 

I can change the dots, or I could remove them completely and instead have a slight bit of the next card showing to prompt a swipe instead. Like shown: 
image

What do you think?

**Comment by cherylli** [comment]
oh yeah that does look terrible. I think the dots are fine, can they be moved up or is the style fixed?

**Pull request merged**